osf/migrations/0001_initial.py

```diff
@@ -2225,16 +2225,6 @@ class Migration(migrations.Migration):
             },
             bases=('osf.abstractprovider',),
         ),
-        migrations.CreateModel(
-            name='QuickFilesNode',
-            fields=[
-            ],
-            options={
-                'proxy': True,
-                'indexes': [],
-            },
-            bases=('osf.abstractnode',),
-        ),
```

Comment:
Is this the right way to do this? Shouldn't we forward-migrate by deleting the model, the way you did in migration 0031, instead of altering the migration history?

Reply:
I am doing both here (and try to remove historical model when I can) because it slows down local and test builds to create table with it's own content_type etc, only to remove it before run time. So it doesn't really effect production, but help speed things up incrementally.  It's a not really strictly necessary here but when you thing that it has to create the model and destroy for each build it's a lot.